convert.go

```diff
@@ -117,7 +117,12 @@ func ToNetAddr(maddr ma.Multiaddr) (net.Addr, error) {
 	case "udp", "udp4", "udp6":
 		return net.ResolveUDPAddr(network, host)
 	case "utp", "utp4", "utp6":
-		return utp.ResolveAddr(network, host)
+		a, err := net.ResolveUDPAddr("udp"+network[3:], host)
+		if err != nil {
+			return nil, err
+		}
+
+		return utp.MakeAddr(a), nil
```

Comment:
i think it makes sense to have this as `utp.ResolveAddr` as before-- there's a sort of "standard interface" for a `net.Conn` protocol and has all these pieces.